webrtc.html

```diff
@@ -4685,7 +4685,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
         <li>
           <p>If the <a href="#dfn-underlying-data-transport">transport</a> was
           closed <dfn id="data-transport-closed-error">with an error</dfn>,
-          fire an NetworkError event at <var>channel</var>.</p>
+          queue a task to fire an NetworkError event at <var>channel</var>.</p>
```

Comment:
(maybe this is what you're saying @jan-ivar ) this change would mean we change the order of the NetworkError and close events.